lib/GenomeFileUtil/core/GenbankToGenome.py

```diff
@@ -473,38 +475,35 @@ def _validate_existing_assembly(self, assembly_ref, genome_obj):
         if len(unmatched_ids_md5s) > 0:
             raise ValueError(warnings["assembly_ref_diff_seq"].format(", ".join(unmatched_ids_md5s)))
 
-    def _load_contigs_for_and_validate_existing_assemblies(self, genome_objs):
```

Comment:
I don't quite understand why this method needed to change - it's run before anything is saved to the storage systems right? Looking at the code, the main thing that needed to happen was the genbank needed to be parsed prior to saving stuff in the blobstore

Reply:
do you understand now? We touched on this topic in previous conversations.

Reply:
I understand why you did things the way you did them, yes
:+1: